staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -95,6 +95,26 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	watchCacheIncreaseTotal = metrics.NewCounter(
```

Comment:
watchCacheSizeIncreaseTotal maybe? [the size explains what it is about]

Reply:
Done, rename it as watchCacheCapacityIncreaseTotal.